lib/vector-core/src/event/vrl_target.rs

```diff
@@ -121,17 +124,20 @@ impl vrl_core::Target for VrlTarget {
                     expected: VALID_METRIC_PATHS_SET,
                 }
                 .to_string())
+                    */
             }
         }
     }
 
-    fn get(&self, path: &LookupBuf) -> std::result::Result<Option<vrl_core::Value>, String> {
+    fn get(&self, path: &LookupBuf) -> std::result::Result<Option<vrl_core::SharedValue>, String> {
         match self {
             VrlTarget::LogEvent(log, _) => log
                 .get(path)
-                .map(|val| val.map(|val| val.clone().into()))
+                .map(|val| val.map(|val| val.clone()))
```

Comment:
The clone here is now a cheap increment of the reference count. We also don't need the `into` since the value is already stored as a `vrl::Value`.